ci-scripts/yocto.groovy

```diff
@@ -135,6 +135,23 @@ void runSmokeTests(String yoctoDir, String imageName) {
     }
 }
 
+void runBitbakeTests(String yoctoDir) {
+    String archiveDir = "bitbake_TestReport"
+
+    stage("Perform Bitbake Testing"){
+        vagrant("/vagrant/cookbook/yocto/run-bitbake-tests.sh ${yoctoDir} ")
+    }
+    
+    //stage("Publish bitbake test results") {
```

Comment:
why do we need commented code?

Reply:
Im testing it. Not sure if publishing the test results is something specific of qemu tests or not.